packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -11,11 +11,17 @@ export const Primary = Template.bind({});
 export const Secondary = Template.bind({});
 Secondary.args = { type: 'secondary' };
 
-export const Medium = Template.bind({});
-Medium.args = { size: 'medium' };
+export const MediumPrimary = Template.bind({});
+MediumPrimary.args = { size: 'medium' };
```

Comment:
Not so sure we are using this right. 
```suggestion
export const MediumPrimary = <Button  size="medium">Call to action</Button>
```

isnt this the same?

Reply:
> Not so sure we are using this right.

I am trying to use what they recommend [here](https://storybook.js.org/docs/react/writing-stories/args). There is a few advantages when using `args`

Reply:
Right okay I see thanks for explaining

Reply:
> isnt this the same?

Well in this case it will throw errors. To go over them we would have to first  to add `onClick={() => {}}` to every Button. And then stop using `args`. And `args` it what allows Storybook and its addons to live edit components.